python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_flooding.py

```diff
@@ -332,209 +350,211 @@ def functionFlooding(X):
 # We can compute a 95% confidence interval of the parameter :math:`\theta^\star`.
 #
 # This confidence interval is based on bootstrap, based on a sample size equal
-# to 100 (as long as the value of the `ResourceMap` key "NonLinearLeastSquaresCalibration-BootstrapSize" is unchanged).
-# This confidence interval reflects the sensitivity of the optimum to the variability in the observations.
+# to 100 (as long as the value of the `ResourceMap` key
+# "NonLinearLeastSquaresCalibration-BootstrapSize" is unchanged).
```

Comment:
```suggestion
# to the value of the `ResourceMap` key
# "NonLinearLeastSquaresCalibration-BootstrapSize".
```

Reply:
Maybe you could add code to print the value of that ResourceMap key? That way the example will remain true if we change the default value of the key down the road.
You did something similar below, btw.

Reply:
This is a good idea. I fixed that and formatted the link to the ResourceMap class.